bidwire/notifiers/cityofboston_notifier.py

```diff
@@ -12,5 +13,16 @@ def get_listings_pre_text(self, bids_length):
         return "We have found " + formatted_text + " since we last sent " + \
             " you an update: "
 
-    def make_item_body(self, bid):
-        return make_bid_item_body(bid)
+    def make_item_body(self, item):
+        return make_bid_item_body(item)
+
+
+class CityOfBostonNoticeNotifier(BaseNotifier):
+    def get_site(self):
+        return Notice.Site.BOSTON
+
+    def make_item_body(self, item):
+        return make_doc_item_body(item)
+
+    def get_listings_pre_text(self, items_length):
```

Comment:
You're missing a return at the end of this file. I suggest setting your editor to make sure this is done automatically for you (some command-line tools will not recognize the last line as a line if it does not end in a return)